### ISSUE TITLE
Can't preselect an item by value in Select or SelectField

### ISSUE THREAD

**Issue by fbitterlin** [ckshekhar73/react-semantic-redux-form#14]
I try to get a SelectField with a certain item preselected. Both options and the selected value are fetched with asnyc redux actions.

I can choose the different options to change the value or just submit the form without changes, eatherway I get the expected value from the form(initialValue or new value). So the value is there.

Only thing that is not working, on initialization no text is shown in the field. No Placeholder is shown and neither is the text of the preselected item. The field stays empty until you choose an option.

This is my SelectField. Value prop is set by Redux and ReduxForm.
```
<Field fluid name='providerId' 
          component={SelectField}
          options={provider}
          placeholder='Choose a Provider' />
```

Is there a way to get it working like in normal react semantic ui, where you just have to set the value prop to preselect an item? I tried fiddling around with the SelectField in your Demo, but I couldn't get it working there either.

```
import React from 'react'
import { Select } from 'semantic-ui-react'

let options = [{value:"1", text:"One"}, {value:"2", text:"Two"}];

const SelectExample = () => (
  <Select value='2' options={options} />
)

export default SelectExample
```

**Comment by ckshekhar** [comment]
@fbitterlin is this resolved?

**Comment by ckshekhar** [comment]
```javascript
import React from 'react'
import { SelectField } from 'react-semantic-redux-form'

let options = [{value:"", text:"Choose a Provider"}, {value:"1", text:"One"}, {value:"2", text:"Two"}];

<Field fluid name='providerId' 
          component={SelectField}
          options={options}
          placeholder='Choose a Provider' />
```
This will preselect item which value is empty.

**Comment by fbitterlin** [comment]
Thanks for your reply. But I since identified semantic-ui-reacts Select as the source of my problem.

Simplified my use case says that the Select should show the users previous selection on this page as selected. This information gets fetched asynchronosly from an api while a loading dimmer is shown to the user. Fetching is dispatched from the componentDidMount-Function in the Component the Select is in. But this means it is first rendered with an empty redux state and thus the value is `undefined` on the first rendering. 

For other input field types it was enough to set `enableReinitialize: true` to repopulate them with the fetched information. But Select from the semantic-ui-react package ignores further changes to the prop after the first render. That's why it stays on `undefined` as the preselected value until you change it by hand in the UI.

Sorry, that I blamed your code for this at first!
